tsconfig.json

```diff
@@ -4,7 +4,8 @@
         "types": [
             "node"
         ],
-        "target": "es2016"
+        "target": "es2016",
+        "experimentalDecorators": true

```

Comment:
correct, TypeScript is trying to *not* invent features but instead implement precisely the features speced out for future EcmaScript releases. Decorators are apparently not fully speced out yet, so they may change, meaning that TypeScript may adjust in the future. Hence this flag, most likely the flag will simply become obsolete (and the behavior activated by default) within the next year.